accessibility/test.js

```diff
@@ -2,30 +2,52 @@ const AxeBuilder = require('@axe-core/webdriverjs');
 const WebDriver = require('selenium-webdriver');
 const firefox = require('selenium-webdriver/firefox');
 
+const baseUrl = `${process.env.FRONTEND_PROTO}://${process.env.FRONTEND_HOST}:${process.env.FRONTEND_PORT}`
```

Comment:
Similarly duplicated :)  - I know that I already did this. So apologies

Reply:
I'm not sure that we would gain much from re-factoring the accessibility tests in the same way the acceptance tests are being done... 

I don't think it makes sense to move the `FirefoxDriver` wrapper class I wrote for acceptance to a higher level (project root), and I don't think we should couple accessibility to acceptance necessarily. Also, in order to use `AxeBuilder`, we need to pass in a Selenium `WebDriver` object.

If it's a sticking point I'm happy to figure something out, but I think for now there isn't a lot of repitition on the accessibility file so I'm going to submit this for re-review!

Reply:
Agreed